lib/pure/os.nim

```diff
@@ -1070,8 +1070,8 @@ proc createHardlink*(src, dest: string) =
   ## Create a hard link at `dest` which points to the item specified
   ## by `src`.
   ##
-  ## **Warning**: Most OS's restrict the creation of hard links to
-  ## root users (administrators) .
+  ## **Warning**: Some OS's restrict the creation of hard links to
```

Comment:
No, it is not. As the commit message says there are no systems which require a root user for hard links. Dropping the complete warning would be fine too.